styles.css

```diff
@@ -9,12 +9,37 @@ body {
 }
 
 .header {
+    padding: 40px 0;
     display: flex;
     justify-content: space-between;
 }
 
 .logo {
-    padding: 40px 0;
+    position: relative;
+    padding: 40px 0px;
+    padding-left: 44px;
+    font: bold 24px/31px pt_serif, arial, helvetica, sans-serif;
+}
+
+.logo__circle {
+    position: absolute;
+    left: 0;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 28px;
+    height: 28px;
+    border-radius: 50%;
+    opacity: .7;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Goal_tereshchenko/pullRequest?prid=6223281)